src/Algebra/Construct/Quotient/Group.agda

```diff
@@ -0,0 +1,140 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Quotient groups
+------------------------------------------------------------------------
+
+{-# OPTIONS --safe --cubical-compatible #-}
+
+open import Algebra.Bundles using (Group)
+open import Algebra.NormalSubgroup using (NormalSubgroup)
+
+module Algebra.Construct.Quotient.Group  {c ℓ} (G : Group c ℓ) {c′ ℓ′} (N : NormalSubgroup G c′ ℓ′) where
+
+open Group G
+
+import Algebra.Definitions as AlgDefs
+open import Algebra.Morphism.Structures
+open import Algebra.Properties.Group G
+open import Algebra.Structures using (IsGroup)
+open import Data.Product.Base
+open import Level using (_⊔_)
+open import Relation.Binary.Core using (_⇒_)
+open import Relation.Binary.Definitions using (Reflexive; Symmetric; Transitive)
+open import Relation.Binary.Structures using (IsEquivalence)
+open import Relation.Binary.Reasoning.Setoid setoid
+
+open NormalSubgroup N
+
+infix 0 _by_
+
+data _≋_ (x y : Carrier) : Set (c ⊔ ℓ ⊔ c′) where
+  _by_ : ∀ g → x // y ≈ ι g → x ≋ y
+
+≋-refl : Reflexive _≋_
+≋-refl {x} = N.ε by begin
+  x // x ≈⟨ inverseʳ x ⟩
+  ε      ≈⟨ ι.ε-homo ⟨
+  ι N.ε  ∎
+
+≋-sym : Symmetric _≋_
+≋-sym {x} {y} (g by x//y≈ιg) = g N.⁻¹ by begin
+  y // x      ≈⟨ ⁻¹-anti-homo-// x y ⟨
+  (x // y) ⁻¹ ≈⟨ ⁻¹-cong x//y≈ιg ⟩
+  ι g ⁻¹      ≈⟨ ι.⁻¹-homo g ⟨
+  ι (g N.⁻¹)  ∎
+
+
+≋-trans : Transitive _≋_
+≋-trans {x} {y} {z} (g by x//y≈ιg) (h by y//z≈ιh) = g N.∙ h by begin
+  x // z              ≈⟨ ∙-congʳ (identityʳ x) ⟨
+  x ∙ ε // z          ≈⟨ ∙-congʳ (∙-congˡ (inverseˡ y)) ⟨
+  x ∙ (y \\ y) // z   ≈⟨ ∙-congʳ (assoc x (y ⁻¹) y) ⟨
+  (x // y) ∙ y // z   ≈⟨ assoc (x // y) y (z ⁻¹) ⟩
+  (x // y) ∙ (y // z) ≈⟨ ∙-cong x//y≈ιg y//z≈ιh ⟩
```

Comment:
Already covered by `Algebra.Properties.Loop`...? And if not, could/should be added?